pkg/contract_watcher/header/converter/converter.go

```diff
@@ -89,9 +89,9 @@ func (c *Converter) Convert(logs []gethTypes.Log, event types.Event, headerID in
 				if len(b) == 32 {
 					seenHashes = append(seenHashes, common.HexToHash(strValues[fieldName]))
 				}
-			case byte:
-				b := input.(byte)
-				strValues[fieldName] = string(b)
```

Comment:
Is there any need to keep this `case byte` in here as well? 🤔

Reply:
I don't think so - `byte` is an alias for `uint8`, so I could probably leave that as the name for this `case` if that seems clearer

Reply:
Gotcha. I like this case as `uint8`, it makes the type check on the next line make more sense to me.